matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -267,12 +275,22 @@ var PhenoTips = (function (PhenoTips) {
             var hasGeneExomeTypeMatch = match.matchingGenesTypes.indexOf(this._filterValues.geneStatus) > -1;
             // by match status (rejected, saved, uncategorized)
             var hasMatchStatusMatch = this._filterValues.matchStatus[match.status];
+
+            var recordOwnershipMatch = function(record) {
+                                         return (this._filterValues.ownerStatus["me"] && record.ownership.userIsOwner)
+                                             || (this._filterValues.ownerStatus["group"] && record.ownership.userGroupIsOwner)
+                                             || (this._filterValues.ownerStatus["public"] && record.ownership.publicRecord)
+                                       };
+            var referenceRecordMatch = match.reference.serverid == "local" && recordOwnershipMatch(match.reference);
```

Comment:
couple problems: 
- not `serverid`, its `serverId`
- for local matches we leave `serverId` just blank `""`